Revision: par2cmdline--mainline--0--patch-1
Archive: dilinger@voxel.net--2004
Creator: Andres Salomon <dilinger@voxel.net>
Date: Wed Jan 14 23:21:45 EST 2004
Standard-date: 2004-01-15 04:21:45 GMT
Modified-files: Makefile.am
New-patches: dilinger@voxel.net--2004/par2cmdline--mainline--0--patch-1
Summary: hardlinks?
Keywords: 

Don't create hard links when installing; use symlinks instead.  Seriously, wtf?

* added files

    {arch}/par2cmdline/par2cmdline--mainline/par2cmdline--mainline--0/dilinger@voxel.net--2004/patch-log/patch-1

* modified files

--- orig/Makefile.am
+++ mod/Makefile.am
@@ -55,9 +55,9 @@
 TESTS = pretest test1 test2 test3 test4 test5 test6 posttest
 
 install-exec-hook :
-	ln -f $(DESTDIR)$(bindir)/par2$(EXEEXT) $(DESTDIR)$(bindir)/par2create$(EXEEXT)
-	ln -f $(DESTDIR)$(bindir)/par2$(EXEEXT) $(DESTDIR)$(bindir)/par2verify$(EXEEXT)
-	ln -f $(DESTDIR)$(bindir)/par2$(EXEEXT) $(DESTDIR)$(bindir)/par2repair$(EXEEXT)
+	ln -sf $(bindir)/par2$(EXEEXT) $(DESTDIR)$(bindir)/par2create$(EXEEXT)
+	ln -sf $(bindir)/par2$(EXEEXT) $(DESTDIR)$(bindir)/par2verify$(EXEEXT)
+	ln -sf $(bindir)/par2$(EXEEXT) $(DESTDIR)$(bindir)/par2repair$(EXEEXT)
 
 uninstall-hook :
 	rm -f $(DESTDIR)$(bindir)/par2create$(EXEEXT)



